CODEOWNERS

```diff
@@ -0,0 +1,309 @@
+geode-core/**/org/apache/geode/cache/*                            @dschneider-pivotal @DonalEvans
+geode-core/**/org/apache/geode/cache/util/**                      @dschneider-pivotal @DonalEvans
+
+#-----------------------------------------------------------------
+# Distributed Locks
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/distributed/internal/locks/**      @kirklund @dschneider-pivotal
+
+#-----------------------------------------------------------------
+# Core region implementations and plumbing
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/*                   @nabarunnag @DonalEvans
+
+#-----------------------------------------------------------------
+# Region entry management
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/entries/**          @dschneider-pivotal @sabbey37
+geode-core/**/org/apache/geode/internal/cache/region/entry/**     @dschneider-pivotal @sabbey37
+geode-core/**/org/apache/geode/internal/cache/map/**              @dschneider-pivotal @sabbey37
+geode-core/**/org/apache/geode/compression/**                     @dschneider-pivotal @sabbey37
+geode-core/**/org/apache/geode/internal/cache/compression/**      @dschneider-pivotal @sabbey37
+
+#-----------------------------------------------------------------
+# Partitioned Regions
+#-----------------------------------------------------------------
+geode-rebalancer/**                                               @boglesby @BenjaminPerryRoss
+geode-core/**/org/apache/geode/internal/cache/partitioned/**      @boglesby @BenjaminPerryRoss
+geode-core/**/org/apache/geode/cache/partition/**                 @boglesby @BenjaminPerryRoss
+
+#-----------------------------------------------------------------
+# Event tracking
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/event/**            @agingade @nabarunnag @gesterzhou
+
+#-----------------------------------------------------------------
+# Eviction
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/eviction/**         @agingade @dschneider-pivotal
+geode-core/**/org/apache/geode/internal/size/**                   @agingade @dschneider-pivotal
+
+#-----------------------------------------------------------------
+# Offheap
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/offheap/**                @pivotal-eshu @dschneider-pivotal
+
+#-----------------------------------------------------------------
+# Transactions
+#-----------------------------------------------------------------
+geode-core/**/org/apache/geode/internal/cache/tx/**               @pivotal-eshu @gesterzhou
```

Comment:
```suggestion
geode-core/**/org/apache/geode/internal/cache/tx/**               @pivotal-eshu @gesterzhou @luissson
```

Reply:
Unfortunately @luissson is not a Geode committer so cannot be a code owner.